lib/fs-driver.js

```diff
@@ -30,7 +30,9 @@ class FsDriver {
     if (!isString(str)) {
       str = str.toString()
     }
-    return str.replace(/(['/\\])/g, '\\$1')
+    return str.replace(/'/g, '\'"\'"\'')
```

Comment:
Can you put a link to that SO question, or some basic explanation above it in comments